### PR TITLE
Document config and program argument changes for wazuh/wazuh#2007

### PR DESCRIPTION
**Pull request by xesxen**
Related PR: https://github.com/wazuh/wazuh/pull/2007

### CODE REVIEW

**Comment by JuanjiJG** [comment]
Hello @xesxen,

Thanks for your contribution.

Please, change the destination branch to `3.8` and make sure there are no conflicts.

Regards,
Juanjo

File: source/user-manual/reference/ossec-conf/auth.rst [hunk 1]
```diff
@@ -208,6 +209,21 @@ When set to ``no``, the maximum limit of agents that can be added is ignored.
...
+
+When set to ``no``, it listens on an IPv4 address.
+
++--------------------+---------------------+
+| **Default value**  | no                  |
++--------------------+---------------------+
+| **Allowed values** | yes, no             |
++--------------------+---------------------+
+
+.. versionadded:: 3.8.0
```
Comment (JuanjiJG) [inline source/user-manual/reference/ossec-conf/auth.rst]: This should be at the beginning of the section:
```rst
ipv6
^^^^

.. versionadded:: 3.8.0

Toggles wether...
```

**Comment by JuanjiJG** [comment]
Adding link to related PR on `wazuh/wazuh` to the first comment of this PR.

**Comment by JuanjiJG** [comment]
This PR should be merged after #642.

**Comment by davidfmiranda** [comment]
Hello @xesxen we appreciate your contribution. Now we are waiting for  [wazuh/wazuh#2007](https://github.com/wazuh/wazuh/pull/2007) to be merged. Once it is done we will merge this PR.

Regards,

David

**Comment by xesxen** [comment]
Closing due to inactivity

**Pull request closed without merging**